### PR TITLE
Allison

### PR DESCRIPTION
**Pull request by atheobold**
Minor changes to Starting with Data, suggested in Issue #76.

### CODE REVIEW

**Review by juanfung — APPROVED** [review]
@atheobold Thank you! I think this addresses `read.csv` clearly and succintly

**Comment by juanfung** [comment]
Holding off on merge while we figure out why build is failing

**Comment by fmichonneau** [comment]
There are 2 GitHub Actions that are being triggered by pull requests in this repository.

The first one ("Website / build-website"), which is failing here, includes running `make lesson-check`. The output of the log is visible here: https://github.com/datacarpentry/r-socialsci/pull/257/checks?check_run_id=1211346820#step:10

The `lesson-check` step looks for things that are not formatted correctly in the lessons. Here, the issues that are identified are:
- the word 'FIXME' appears in the files `reference.md` and `_extras/discuss.md`
- the file name for the lesson about JSON is not standard (it should start with a double-digit)
- there is a blockquote of class "Warning" in the `{dplyr}` lesson that is not being recognized (this is a false positive that was just fixed in the styles repo)
- there is a blockquote paragraph in the `_extras/guide.md` that is not associated with any of the known styles

You should be able to reproduce this locally by running `make lesson-check`.

These issues are things that can lead to the output of the lesson to not look like they should but generally are not things that break the rendering of the lessons. This is indicated by the second GitHub Actions ("Build and Deploy Website") passing successfully.

In other words, the changes introduced by this pull request are valid and the website builds without error. This pull request could be merged as is. The issues highlighted by the first GitHub Actions existed before this pull request and could be fixed separately.

Don't hesitate to reach out if I can help or clarify things further. Thank you!

**Comment by juanfung** [comment]
@fmichonneau Thank you, your explanation is very helpful!
